src/test/experiments.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { expect } from "chai";
+import { enableExperimentsFromCliEnvVariable, isEnabled } from "../experiments";
+
+describe("experiments", () => {
+  let originalCLIState = process.env.FIREBASE_CLI_EXPERIMENTS;
+
+  before(() => {
+    originalCLIState = process.env.FIREBASE_CLI_EXPERIMENTS;
+  });
+
+  beforeEach(() => {
+    process.env.FIREBASE_CLI_EXPERIMENTS = originalCLIState;
+  });
+
+  afterEach(() => {
+    process.env.FIREBASE_CLI_EXPERIMENTS = originalCLIState;
```

Comment:
This still leaves the experiments enabled that are manipulated in the test. Should this run through all the experiments and reset them to not enabled, then call `enableExperimentsFromCliEnvVariable` after resetting the env var?

Reply:
True. I decided to just manually reset "experiments" in the test. 🤷